python/mxnet/gluon/nn/activations.py

```diff
@@ -158,7 +158,7 @@ def __init__(self, alpha=1.0, **kwargs):
         self._alpha = alpha
 
     def hybrid_forward(self, F, x):
-        return F.where(x > 0, x, self._alpha * (F.exp(x) - 1.0))
+         F.LeakyReLU(x, act_type='elu', slope=self._alpha)
```

Comment:
Seems like you have an extra whitespace here, please get rid of that and this is good to go.

Reply:
Sorry, less `return`.